### ISSUE TITLE
Move buildscripts into subdirectories

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#73]
## User story
1. As a Software Developer
2. I want to move build scripts into subdirectories
3. So that the project structure is more organized and easier to maintain

## Acceptance criteria
* .bat files are moved to the corresponding subdirectories (WNDOWS_GUI and TESTING_WINDOWS_GUI) and removed from the root directory 
* Update documentation on how to build / run the app

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
